plugins/wazo-gigaset/N510/templates/base.tpl

```diff
@@ -113,6 +113,8 @@
      Redirect device always to own provisioning Server so you are in control for example: Firmware updates 	-->
     {%- if http_port %}
     <SYMB_ITEM ID="BS_IP_Data1.aucS_DATA_SERVER[0]" class="symb_item" value='"{{ ip }}:{{ http_port }}"'/>
+    <SYMB_ITEM ID="BS_IP_Data1.aucS_USER_FIRMWARE_URL" class="string" value='"{{ ip }}:{{ http_port }}/42/fw262.bin"' />
```

Comment:
Still according to the documentation, `S_SPECIAL_DATA_SRV` has higher priority so if it is not empty then `S_DATA_SERVER` and `S_USER_FIRMWARE_URL` will be ignored.
So I think these 2 last additional parameters may be removed.